tasks/mutation-testing.js

```diff
@@ -115,6 +115,11 @@ function createTestsFailWithoutMutationsLogMessage(opts, srcFilePath) {
     return srcFileName + ' tests fail without mutations';
 }
 
+function createTestsFailWithoutMutationsLogMessage(opts, srcFilePath) {
+    var srcFileName = createMutationFileMessage(opts, srcFilePath);
+    return srcFileName + ' tests fail without mutations';
+}
```

Comment:
I only noticed this after merging: it looks like this method has been duplicated. Was that on purpose?